messaging/WFLY-11829_artemis_clusters_with_http_load_balancer.adoc

```diff
@@ -0,0 +1,68 @@
+= Support messaging clusters (Artemis Clusters) behind http load balancers
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          messaging,jms,load-balancer
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11829
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ENTMQBR-1001
+* https://issues.apache.org/jira/browse/ARTEMIS-1322
+* https://issues.apache.org/jira/browse/EAP7-704
+* https://issues.jboss.org/browse/EAP7-669
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+== Requirements
+
+* Add a boolean `use-topology-load-balancing` attribute, with a default value set to `true`, to 
```

Comment:
https://github.com/ehsavoie/wildfly/commit/966abbf56d0f5483f0360f11757ea00a69349293 this should be `use-topology-for-load-balancing`, could you please update here?